src/main/flight/pid.c

```diff
@@ -216,6 +219,18 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     } else {
         dtermNotchFilterApplyFn = nullFilterApply;
     }
+    
+    //extra Dterm PT1 filter
+	static pt1Filter_t ExtraPt1FilterD;
+	if (pidProfile->dterm_lpf_2_hz == 0 || pidProfile->dterm_lpf_2_hz > pidFrequencyNyquist) {
+		dtermLpf2ApplyFn = nullFilterApply;
```

Comment:
better to set this regardless and then do the following block in the if(){} instead of if/else.